src/components/PupilComponents/QuizResultQuestionStem/QuizResultQuestionStem.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  OakCloudinaryImage,
+  OakFlex,
+  OakSpan,
+} from "@oaknational/oak-components";
+
+import {
+  removeMarkdown,
+  shortAnswerTitleFormatter,
+} from "@/components/TeacherComponents/LessonOverviewQuizContainer/quizUtils";
+import {
+  ImageItem,
+  TextItem,
+} from "@/node-lib/curriculum-api-2023/queries/pupilLesson/pupilLesson.schema";
+import { getSizes } from "@/components/SharedComponents/CMSImage/getSizes";
+
+export type QuizQuestionStemProps = {
```

Comment:
If we're going to have a shared type for props between two components then they need to be stored in a higher order folder. But actually I don't think it's appropriate here.

Reply:
ok..